configurations/mt-staging.ts

```diff
@@ -291,6 +291,16 @@ const preEngagementConfig: PreEngagementFormDefinition = {
         },
       ],
     },
+    {
+      type: 'checkbox',
+      name: 'termsAndConditions',
+      label:
+        'I have read and accept the Terms and Conditions https://kellimni.com/chat-terms/',
```

Comment:
You are missing the href link here... se how it is done for CL:
  'He leído y acepto los <a href="https://www.linealibre.cl/wp-content/uploads/2020/11/TERMINOS-Y-CONDICIONES-DE-USO-Y-POLITICA-DE-PRIVACIDAD-LL.pdf">términos y condiciones</a>',